src/OgGroupAudienceHelper.php

```diff
@@ -24,6 +24,15 @@ class OgGroupAudienceHelper {
    */
   const DEFAULT_FIELD = 'og_group_ref';
 
+  /**
+   * The name of the field which reference user to groups.
+   */
+  const USER_REFERENCE_FIELD = 'og_membership_reference';
+
+  /**
+   * The name of the field which reference non-user entities to groups.
+   */
+  const NON_USER_REFERENCE_FIELD = 'og_standard_reference';
```

Comment:
It's a bit weird to call this the "non-user" reference field. This is after all the default one, calling it non-user makes it sound like it is some kind of an exception. Maybe `REFERENCE_FIELD_STANDARD` would be better?

Again, we should actually get rid of the user referencing field, then this can become simply `OG_REFERENCE_FIELD`.